src/skip-wip-ci/skip-wip-ci.yml

```diff
@@ -0,0 +1,62 @@
+# Orb Version 0.1.0
+
+version: 2.1
+description: 'Use to skip CI when PR title contains "wip", "skip ci", "ci skip" or is draft'
+
+commands:
+  check-skippable-pr:
+    steps:
+      - run:
+          shell: /bin/bash
```

Comment:
I think the default shell is bash so this _shouldn't_ be necessary.

Reply:
I guess it might not be depending on the execution environment... and because you're executing this in an `alpine` instance, it could be something else. 

I stand corrected 👍